### DISCUSSION TITLE
Release (beta): 1.1-beta11

### DISCUSSION THREAD

**Discussion by apohl79** [apohl79/audiogridder#316] [General]
Hey all,

I just uploaded the **1.1-beta11** release. Here is a few highlights:

- Experimental support for Macs with an Apple Silicon (arm64) chip
- Plugin search feature
- Other improvements and fixes

Please check out the [release page](https://github.com/apohl79/audiogridder/releases/tag/release_1_1_beta11) for more details and the downloads.

For those who have issues with the installers on Windows or macOS, I also provide the zipped binaries from now on.

Cheers,
Andreas

**Comment by kvltst** [comment]
I have an Apple MBP M1 model. I'll look at your build when I get chance. Cheers.

**Reply by rockumk** [reply]
Same here.

**Reply by apohl79** [reply]
thats great!

**Reply by kvltst** [reply]
M1 Server / Plugin Installers worked as expected (request access, approving app in security, etc).

I don't have any VST currently installed for M1 specifically, but Server App is running. Will find some VSTs to emulate later.

This may be a great use for the new M1 Mac Mini. Power efficient, DSP Farm. :)

**Reply by apohl79** [reply]
This is great news! You might be able to test with the apple audio units, that should always be present.

**Comment by kastorjunior** [comment]
Works great for me on Win10 with a Asus laptop (M2 hardrive+16 gigas ram) 1.8 ghz, 3.9 ghz on turbomode (usbc to Rj45) and a PC  with an I 9900k oC to 5ghz + 64 gigas ram. 2 M2 drives.
mobo : z390 designare with 2 etrhenet ports. I wil by a second hand dell server.
keep up the good work. You'll receive 10 euros from franck. THX

**Reply by CaBuffalo** [reply]
Nice !!

I have it tested in two environment.
1, Have windows as server in Parralles running in Catalina. No issue.
2, Have two win 10 machines running as server and MacOS Catalina as DAW. Very stable. Only had a retart of server when I quit my session. I already sent the dump file to Andreas.

Looking forward to the final ver. :)

**Comment by sammbusy** [comment]
Should i uninstall anything before installing this ? (both windows and mac platforms)

**Reply by CaBuffalo** [reply]
No, you don't need to do that.

**Reply by sammbusy** [reply]
thank you !

**Comment by dsstudio1** [comment]
What you are doing is great!!! If you have time, pay attention to the Linux distribution kit AV Linux http://www.bandshed.net/avlinux This distribution is very musician-friendly. If you could compile plugins for it it would be great!!!

**Comment by TheSyndrome-2020** [comment]
I also concur with this

Well Done Andreas

**Comment by gormanj** [comment]
Great software. I get a video stream when using stable 1.0. No video window is opened when I use 1.1 b11 in minihost modular, plugin doesn't connect to server using NI Komplete Kontrol. Everything seems to be working fine in Logic. I'm using Mac OS X 10.13.6 I have two displays running from a GTX 1080. I've dried the different video protocols on my Win10 that is running the server.

**Comment by CaBuffalo** [comment]
Do you mean video streaming as the plugin window mirroring?

That's great to see that it works for you.

On Wed, Jan 20, 2021 at 7:40 PM gormanj <notifications@github.com> wrote:

> Great software. I get a video stream when using stable 1.0. No video
> window is opened when I use 1.1 b11. Everything other than that seems to
> work great. I'm using Mac OS X 10.13.6 I have two displays running from a
> GTX 1080. I've dried the different video protocols on my Win10 that is
> running the server.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/316#discussioncomment-298003>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BPABBQMVUJ2WDFD5TS26O3RANCNFSM4WHK7KIQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by gormanj** [reply]
> Do you mean video streaming as the plugin window mirroring? That's great to see that it works for you.

Yes, but I updated my post to include more info. Works perfectly in Logic!
